### ISSUE TITLE
[Documentation] serial protocol quick summary

### ISSUE THREAD

**Issue by unreal4me** [nicsure/QuanshengDock#86]
Hello there, very nice app and very nice use case!
It will be helpful if there is a semi documentation about the implementation of the CAT serial protocol. Like for example does it work with hamlib/rigctl etc?

**Comment by BranoSundancer** [comment]
TL;DR: There is partial* emulation of **FT-991**, which is model **1035** in hamlib. This is the command I've been using with QD:
`rigctld.exe -m 1035 -r COM11 -s 38400 1 -vvvv`
The COM11 should be most probably set as `CAT Port` in QD's settings. And then it should work in the **XVFO mode**.

*The partial CAT implementation means not all CAT commands are emulated. Only those, which are required by **gpredict** connected to **rigctld**. See to PoC in the [Facebook group](https://www.facebook.com/groups/289656334131909/posts/364416183322590/). Maybe some other need to be added to support other programs.

**Comment by BranoSundancer** [comment]
Here in [PTT.cs](https://github.com/nicsure/QuanshengDock/blob/103acd3f83ae0d920abfd38e0cd1ef242a9b8451/QuanshengDock/Serial/PTT.cs#L72)

**Comment by unreal4me** [comment]
Thank you very much for your time Bruno, happy coding!

**Comment by BranoSundancer** [comment]
I am not 100% sure but I think I needed to create a virtual COM cable using [com0com](https://com0com.sourceforge.net/) utility. Rigctld was connected to COM11 (as seen above), QD was connected to virtual port COM10 (see screenshot below):

<img width="135" height="50" alt="Image" src="https://github.com/user-attachments/assets/b4085970-d2e2-4ee3-bdc7-efa835f501b9" />

<img width="451" height="391" alt="Image" src="https://github.com/user-attachments/assets/747e4a25-6c9f-4613-9f08-7c01f2c14af3" />
